source/projects/private-voice-assistant/voice_remote_local_device.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: "Setting up a fully local voice remote device"
```

Comment:
Let's remove this file. We have instructions on how to create a local pipeline and instructions on how to setup a device to talk to pipelines.